library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt

```diff
@@ -118,7 +119,16 @@ class DonutProgressView @JvmOverloads constructor(
             invalidate()
         }
 
-    private var entries = mutableListOf<DonutProgressEntry>()
+    private val defaultColor: Int
+        get() {
+            return try {
+                ContextCompat.getColor(context, R.color.colorPrimary)
+            } catch (e: Resources.NotFoundException) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/212ed880cccf1196397e5b12a6300bab95c4049d/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L126"></span>The caught exception is swallowed. The original exception could be lost.</td>
    </tr>
  </tbody>
</table>